src/lib/utils.ts

```diff
@@ -137,3 +137,13 @@ export function assertUnreachable(x: never): never {
 export function assertIsDefined<T>(x: T | undefined | null): asserts x is T {
   if (x === undefined || x === null) throw Error(`unexpected undefined or null`)
 }
+
+/**
+ * https://gist.github.com/hyamamoto/fd435505d29ebfa3d9716fd2be8d42f0?permalink_comment_id=4261728#gistcomment-4261728
+ * only used to anonymize user data - don't user for anything security critical
+ */
+export const hashCode = (str: string): string =>
```

Comment:
@reallybeard up to you what you want hashes to look like in mixpanel - this outputs numbers. whatever is *easiest* to work with